### DISCUSSION TITLE
Highlighted thumbnails

### DISCUSSION THREAD

**Discussion by edubeckha** [igordanchenko/yet-another-react-lightbox#266] [Q&A] [answered]
Good evening,

Is there any way to highlight the thumbnail that is being shown on the gallery? Maybe with a different style just to one this thumbnail image?

Thanks for your attention!

**Comment by igordanchenko** [comment, marked as answer]
Duplicate #207

**Reply by edubeckha** [reply]
Thanks Igor!
